src/containers/List/ListFilters/OptionFilters.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
```

Comment:
Any particular reason why these 3 types are together in one file, but Boolean Filters has its own file?

Reply:
FWIW -- I found the term "optionsFilters" quite confusing until I looked through it properly, so might be nicer just to have one "FilterTypes.tsx" file, or put them all seperate?

Reply:
they kind of all have the same UI, and set of options, and boolean behaves different. I am not happy with the name 'options', happy to change if there is anything better.  Can't think of a better term atm